templates/repo/issue/search.tmpl

```diff
@@ -8,7 +8,8 @@
 		<input type="hidden" name="assignee" value="{{$.AssigneeID}}">
 		<input type="hidden" name="poster" value="{{$.PosterID}}">
 		<input name="q" value="{{.Keyword}}" placeholder="{{.locale.Tr "explore.search"}}...">
-		<button class="ui small icon button" type="submit" aria-label="{{.locale.Tr "explore.search"}}">
+		<button id="hashtag-button" class="ui small icon button gt-hidden" type="submit" aria-label="{{.locale.Tr "explore.goto"}}">{{svg "octicon-hash"}}</button>
```

Comment:
Using `data-tooltip-content="{{.locale.Tr "explore.goto"}}"` is enough (no need to write aria-label)